index.html

```diff
@@ -418,6 +418,16 @@ <h2>Success Criteria</h2>
         <p>
           There should be testing plans for each specification, starting from the earliest drafts.
         </p>
+	<p>
+	In order to advance to Proposed Recommendation, each normative specification must have an open test suite
+	that tests every feature defined in the specification.
+	Additionally, two or more implementaitons should demonstrate interoperability for each feature
```

Comment:
```suggestion
	Additionally, two or more normative implementations should demonstrate interoperability for each feature
```

Reply:
I agree with the spelling fix, but am not totally sure what the difference is between an implementation and a normative implementation.

Reply:
I would have had the same question... We do not really have the term "normative implementation". VC-s are normative data structures/formats whose exchange between implementations must be demonstrated; we do not specify the implementations themselves.